src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
+      var id = ++workers;
+      var startTime = Date.now();
+
+      // Bound helper function for sending worker breadcrumbs
+      //
+      // If IE10 supported WeakMap this could be done by enhancing the prototype, not
+      // the instance's methods directly.
+      function trackWorkerBreadcrumb(name, metaData) {
+        if (metaData.message) {
+          metaData.message = serialiseGenericObject(metaData.message);
+        }
+        self.leaveBreadcrumb({
+          type: "process",
+          name: name,
+          metaData: merge({
+            id: id
+          }, metaData)
+        });
+      }
+
+      trackWorkerBreadcrumb("Created worker", {
+        url: url
+      });
+
+      this.addEventListener("message", function(event) {
+        // TODO Handle automatic error catching from workers
+
+        trackWorkerBreadcrumb("Message from worker", {
+          message: event.data
+        });
+      });
+
+      // Warning: Calling the `postMessage` or `terminate` prototype methods directly
+      // against the worker object will bypass these breadcrumbs.
+
+      enhance(this, "postMessage", function(message) {
+        trackWorkerBreadcrumb("Message to worker", {
+          message: message
+        });
+      });
+
+      enhance(this, "terminate", function() {
+        trackWorkerBreadcrumb("Terminated worker", {
+          uptime: (Date.now() - startTime) + "ms"
```

Comment:
All browsers which support workers, support `Date.now()`, but I can change it for consistency sake?